app/models/manageiq/providers/redhat/infra_manager/refresh/strategies/api3.rb

```diff
@@ -1,4 +1,37 @@
 module ManageIQ::Providers::Redhat::InfraManager::Refresh::Strategies
   class Api3 < ManageIQ::Providers::Redhat::InfraManager::Refresh::Refresher
+    def host_targeted_refresh(inventory, target)
+      methods = {
+        :primary => {
```

Comment:
We have a bug with host targeted refresh and we are missing:

          :network => {:networks => "network" },